### PR TITLE
Add smtp docker image to catch all emails

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by pmauduit** [comment]
Sounds good !

**Comment by pmauduit** [comment]
Just a remark: we might want to move ldap, postgresql and this new smtp/ directory right into docker/, because it is not directly related to the geOrchestra codebase

File: smtp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+RUN useradd -ms /bin/bash -p PcdO6g4gV662A smtp
+
+# Generate script to run at startup
+RUN echo '#!/bin/bash' > /root/start.sh
+RUN echo "/etc/init.d/courier-authdaemon start" >> /root/start.sh
+RUN echo "/etc/init.d/courier-imap start" >> /root/start.sh
+RUN echo "smtp-sink -c -d /home/smtp/Maildir/new/%M. -u smtp 0.0.0.0:25 10" >> /root/start.sh
+RUN chmod +x /root/start.sh
+
+# Run the rest of the commands as the ``postgres`` user
```
Comment (fvanderbiest) [inline smtp/Dockerfile]: postgres, really ? ;-)

Comment (pmauduit) [inline smtp/Dockerfile]: Sorry I merged a bit fast, but since the destination branch is already a WIP branch ... :)

**Pull request merged**